### PR TITLE
fix(client): Sign out messages disappear when user starts typing

### LINKED ISSUE #2211: Sign out messages should disappear on forms when user start logging in
When the user enters characters into the username or password field, the messages should disappear.

fxa-signed-out-successfully
fxa-session-expired

### PR DESCRIPTION
**Pull request by larissagaulia**
PR to fix #2211 + unit test

### CODE REVIEW

File: app/tests/spec/views/form.js [hunk 1]
```diff
@@ -103,6 +103,14 @@ define(function (require, exports, module) {
         assert.isFalse(view.isErrorVisible());
       });
 
+      it('hides messeges when input value is changed', function () {
```
Comment (vladikoff) [inline app/tests/spec/views/form.js]: typo: `messeges` -> `messages`

**Comment by vladikoff** [comment]
@larissagaulia Hey, you don't have to open new pull requests for the same issue. you can keep working in this pull request by just pushing new commits to it. See this guide here: https://www.atlassian.com/git/tutorials/making-a-pull-request/example and more here https://www.atlassian.com/git/tutorials/syncing/git-push

Once we are happy with all the commits we "squash" all changes into one commit. Guide here: https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase-i

File: app/tests/spec/views/form.js [hunk 2]
```diff
@@ -103,6 +103,14 @@ define(function (require, exports, module) {
...
+        view.displayError('this is an error');
+        assert.isTrue(view.isErrorVisible());
+        view.$('#email').val('some@email.com');
+        view.enableSubmitIfValid();
+        assert.isFalse(view.isErrorVisible());
+      });
```
Comment (vladikoff) [inline app/tests/spec/views/form.js]: We also need a test for `displaySuccess`. You can either reuse this same test that you added or create a new one below. 

Here's an existing test that might be helpful to you:

```js

      it('hides any previously displayed success messages', function () {
        view.displaySuccess('the success message');
        view.displayError('the error message');
        assert.isTrue(view.isErrorVisible());
        assert.isFalse(view.isSuccessVisible());
      });
```

File: app/scripts/views/form.js [hunk 3]
```diff
@@ -112,8 +112,9 @@ define(function (require, exports, module) {
         return;
       }
 
+      this.hideError();
```
Comment (vladikoff) [inline app/scripts/views/form.js]: above here could you add a comment, something like this:

```js
// hide success and error messages after user changes the form
```

**Comment by larissagaulia** [comment]
Oh, thanks for the feedback ! If anything else needs improvements just let me know :+1:

**Comment by vladikoff** [comment]
@larissagaulia Awesome! This looks ready to ship, just squash your commits into one using `git rebase` and force push to this branch.

Some guides here: http://push.cwcon.org/learn/squashing-commits.html 

Let me know if you have questions

**Comment by larissagaulia** [comment]
@vladikoff let me know if anything else needs to be changed =)

**Comment by vladikoff** [comment]
@larissagaulia great work! PR Merged :+1:

**Pull request merged**